packages/batch-poster-monitor/index.ts

```diff
@@ -221,23 +219,31 @@ const getBatchPosterLowBalanceAlertMessage = async (
     }
   }
 
-  const balance = await parentChainClient.getBalance({
+  const currentBalance = await parentChainClient.getBalance({
     address: batchPoster,
   })
 
-  const lowBalanceDetected =
-    (childChainInformation.parentChainId === 1 &&
-      balance < BigInt(LOW_ETH_BALANCE_THRESHOLD_ETHEREUM * 1e18)) ||
-    (childChainInformation.parentChainId !== 1 &&
-      balance < BigInt(LOW_ETH_BALANCE_THRESHOLD_ARBITRUM * 1e18))
+  // get the gas used in the last 24 hours
+  let gasUsedInLast24Hours = BigInt(0)
+  for (const log of sequencerInboxLogs) {
+    gasUsedInLast24Hours = (
+      await parentChainClient.getTransactionReceipt({
+        hash: log.transactionHash,
+      })
+    ).gasUsed
+  }
+  const currentGasPrice = await parentChainClient.getGasPrice()
+  const balanceSpentIn24Hours = gasUsedInLast24Hours * currentGasPrice
+
+  const lowBalanceDetected = currentBalance < 2n * balanceSpentIn24Hours // 2 days worth of balance
```

Comment:
Can we move the `2` to a constant? I imagine this is something that other teams might want to easily adjust.